sync/sync.go

```diff
@@ -205,6 +254,9 @@ func (s *Synchronizer) verifierTask(ctx context.Context, block *core.Block, stat
 					// revert the head and restart the sync process, hoping that the reorg is not deep
 					// if the reorg is deeper, we will end up here again and again until we fully revert reorged
 					// blocks
+					if s.plugin != nil {
```

Comment:
you don't need this check because it's already inside `s.handlePluginRevertBlock()`

Reply:
Yup, thanks